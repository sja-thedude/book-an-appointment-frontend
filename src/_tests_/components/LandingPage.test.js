import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../../redux/Store';
import LandingPage from '../../components/landingPage/LandingPage';

describe('Error Page', () => {
  it('renders correctly', () => {
    const tree = render(<Provider store={store}><Router><LandingPage /></Router></Provider>);
    expect(tree).toMatchSnapshot();
  });
});
