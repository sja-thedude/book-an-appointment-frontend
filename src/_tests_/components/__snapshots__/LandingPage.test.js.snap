// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Page renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="landingPage"
      >
        <div
          id="ReactBackgroundSlider"
        >
          <figure
            style="background-image: url(landing1.jpg); animation-delay: 0s;"
          />
          <figure
            style="background-image: url(landing2.jpg); animation-delay: 12s;"
          />
        </div>
        <div
          class="landing"
        >
          <div
            class="image"
          >
            <img
              alt="logo"
              src="logo_transparent.png"
            />
          </div>
          <h1>
            Welcome Back
          </h1>
          <h3>
            Login to your account
          </h3>
          <a
            href="/login"
            style="text-decoration: none;"
          >
            <button
              class="login"
              type="button"
            >
              Login
            </button>
          </a>
          <div
            class="sousTitle"
          >
            <span />
            <h4>
              MY DOCTOR
            </h4>
            <span />
          </div>
          <p>
            Don't have an account?
             
            <a
              href="/register"
              style="text-decoration: none;"
            >
              Register
            </a>
          </p>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="landingPage"
    >
      <div
        id="ReactBackgroundSlider"
      >
        <figure
          style="background-image: url(landing1.jpg); animation-delay: 0s;"
        />
        <figure
          style="background-image: url(landing2.jpg); animation-delay: 12s;"
        />
      </div>
      <div
        class="landing"
      >
        <div
          class="image"
        >
          <img
            alt="logo"
            src="logo_transparent.png"
          />
        </div>
        <h1>
          Welcome Back
        </h1>
        <h3>
          Login to your account
        </h3>
        <a
          href="/login"
          style="text-decoration: none;"
        >
          <button
            class="login"
            type="button"
          >
            Login
          </button>
        </a>
        <div
          class="sousTitle"
        >
          <span />
          <h4>
            MY DOCTOR
          </h4>
          <span />
        </div>
        <p>
          Don't have an account?
           
          <a
            href="/register"
            style="text-decoration: none;"
          >
            Register
          </a>
        </p>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
